Minutes.java

```diff
@@ -1,13 +1,28 @@
 class Minutes extends ConsoleProgram {
 
   /**
-  * Description
-  * @author:
+  * A program which takes an input in minutes and converts it to days, hours, and minutes
+  * @author: Austin Lian
   */
   
   public void run() {
     
-    // start coding here
+    // Initialize variables
+    int intMinutes;
+    int intDays;
+    int intHours;
+    int intMinutesRemainder;
+
+    // Get inputs (Minutes)
+    intMinutes = readInt("Enter the number of minutes you want to convert: ");
+
+    // Calculate Days, Hours, and Minutes
+    intDays = intMinutes / 1440;
+    intHours = (intMinutes % 1440) / 60;
+    intMinutesRemainder = intMinutes % 60;
```

Comment:
Calculates intMinutesRemainder in one go rather then multiple calculations as found in the solutions